jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ContainerParameters.java

```diff
@@ -112,7 +112,6 @@ public void setExtraClasspath(List<String> classpath) {
   }
 
   @Input
-  @Optional
   public boolean getExpandClasspathDependencies() {
```

Comment:
I guess this fixes #2930. And I was wondering if we should fix it by converting it to a `Property` [as we did for `allowInsecureRegistries`](https://github.com/GoogleContainerTools/jib/blob/cc990bec2d5813e86a39a16da655f601c7d5604b/jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java#L95).

Reply:
Perhaps, but I think it's beyond the scope of this PR. (this just fixes the issue that a boolean can't be optional)

Reply:
Okay, I think this is a simple fix that works.